src/features/query-bar/components/filter/FilterButton.js

```diff
@@ -25,7 +25,7 @@ type State = {
 };
 
 type Props = {
-    columns?: Array<ColumnType>,
+    columns: Array<ColumnType>,
```

Comment:
I believe this is optional. 

It is reasonable to render a QueryBar without a template selected which means columns is undefined.